src/token_position.test.ts

```diff
@@ -122,6 +122,40 @@ describe('token_positions', () => {
             }
         })
 
+        it('finds the correct token (without tokenization)', () => {
+            const content = `<span role="presentation" style="padding-right: 0.1px;"><span class="cm-tab" role="presentation" cm-text="	">    </span><span class="cm-keyword">if</span> <span class="cm-variable">rv</span> :<span class="cm-operator">=</span> <span class="cm-variable">contextGet</span>(<span class="cm-variable">r</span>, <span class="cm-variable">varsKey</span>); <span class="cm-variable">rv</span> <span class="cm-operator">!=</span> <span class="cm-atom">nil</span> {</span>`
+
+            // Each offset is 3 more than the corresponding offset in the
+            // tokenized test above because this test case comes from Bitbucket
+            // where tabs are converted to spaces.
+            //
+            // The '(' and ' ' tokens are absent from this test because, on
+            // Bitbucket, punctuation characters are not wrapped in tags and the
+            // current offset-finding logic can't determine the offset for such
+            // tokens. One way to fix that is to use the CodeMirror API
+            // directly.
```

Comment:
So we do need tokenization after all? I thought CodeMirror wrapped all tokens?

Reply:
As far as I can tell, only punctuation characters are not wrapped. Normal tokens are wrapped. Since that covers 99% of use cases, I think it's ok. WDYT?

Reply:
👍